pyproject.toml

```diff
@@ -57,7 +57,7 @@ phonopy_reader = ["h5py>=3.6.0", "PyYAML>=6.0"]  # Deprecated, will be removed i
 phonopy-reader = ["h5py>=3.6.0", "PyYAML>=6.0"]
 brille = ["brille>=0.7.0"]
 test = ["mock", "pytest~=7.0", "coverage", "pytest-mock", "pytest-lazy-fixture", "pytest-xvfb", "python-slugify"]
-ci = ["tox==4.32.2"]
+ci = ["tox==4.23.2"]
```

Comment:
Why the downgrade?

Reply:
4.32 doesn't exist yet, this must have been a typo